common/config/src/lib.rs

```diff
@@ -151,12 +151,21 @@ pub struct K8sStartupLeaseConfig {
     pub option: String,
 }
 
+const LOGDNA_PREFIX: &str = "LOGDNA_";
+const MEZMO_PREFIX: &str = "MZ_";
+
 impl Config {
+    pub fn new_from_env() -> Result<Self, ConfigError> {
+        Config::process_logdna_env_vars();
```

Comment:
A bit confused why this new constructor wrapper calls `process_logdna_env_vars()` when it looks like the old `new()` constructor calls it as well?

Reply:
leftover. thanks!